dropkick/src/main/java/life/genny/dropkick/live/data/InternalConsumer.java

```diff
@@ -136,25 +136,39 @@ public void getEvent(String event) {
 		BaseEntity target = null;
 		List<BaseEntity> definitions = new ArrayList<>();
 		BaseEntity definition = null; // For wip usage
-
+    
 		if (!StringUtils.isBlank(processId)) {
 			ProcessData processData = qwandaUtils.fetchProcessData(processId);
 			if (processData == null) {
 				log.error("Process data not found for processId: " + processId);
 				return;
 			}
-			target = qwandaUtils.generateProcessEntity(processData);
+      target = beUtils.getBaseEntity(processData.getTargetCode());
+      // TODO: This will break if it is an old target without a processId
+      BaseEntity def = defUtils.getDEF(target);
+      definitions.add(def);
+
+			BaseEntity processEntity = qwandaUtils.generateProcessEntity(processData);
+      // update transient target with process enity information
+      log.debug("Adding processentity attributes to target");
+      for (EntityAttribute ea : processEntity.getBaseEntityAttributes()) {
+        EntityAttribute copy = ea;
```

Comment:
This is not a copy.

Reply:
Would be better to `EntityAttribute cope = ea.clone();` or whatever method we have defined for deep-copying EntityAttributes/Coded Entities
target.addAttribute(copy) will link these two base entities to the same entity attribute

Reply:
updated